vsphere_plugin_common/constants.py

```diff
@@ -21,7 +21,9 @@
 
 NETWORK_NAME = 'network_name'
 SWITCH_DISTRIBUTED = 'switch_distributed'
-NETWORK_RUNTIME_PROPERTIES = [NETWORK_NAME, SWITCH_DISTRIBUTED]
+# If you change the next line you will probably break NSX integration
+NETWORK_ID = 'vsphere_network_id'
+NETWORK_RUNTIME_PROPERTIES = [NETWORK_NAME, SWITCH_DISTRIBUTED, NETWORK_ID]
```

Comment:
Honestly? Copying the existing approaches.

Could change them to be tuples... it's probably the case that none of them should be modified, but whether it's the case that none of them are modified I don't know. Perhaps best to consider this for part of the cleanup effort (since we should probably also change the type at the end)?